pixi.toml

```diff
@@ -0,0 +1,94 @@
+[project]
+name = "imod-python"
+version = "0.15.1"
+description = "Make massive MODFLOW models!"
+authors = ["Deltares <huite.bootsma@deltares.nl>", ]
```

Comment:
Something to pick up in a later issue: what to do with this mail address in our aim to "professionalize" the software suite: Add a complete list of authors with full names, like [xmipy](https://github.com/Deltares/xmipy/blob/develop/pyproject.toml) & [hydromt](https://github.com/Deltares/hydromt/blob/main/pyproject.toml)? Or use imod.info@deltares.nl or something, like [ribasimpy](https://github.com/Deltares/Ribasim/blob/main/python/ribasim/pyproject.toml)?